flink-connectors/flink-connector-files/src/test/java/org/apache/flink/connector/file/table/FileSystemOutputFormatTest.java

```diff
@@ -282,7 +278,7 @@ private FileSystemOutputFormat<Row> createSinkFormat(
         TableMetaStoreFactory msFactory = new FileSystemCommitterTest.TestMetaStoreFactory(path);
         return new FileSystemOutputFormat.Builder<Row>()
                 .setMetaStoreFactory(msFactory)
-                .setPath(path)
+                .setStagingPath(new Path(stagingPath.toString()))
```

Comment:
You can still stick to `setPath(stagingBasePath)` here and remove the `stagingPath` field. The only thing you have to change is that you have to assert for empty directory rather than deleted directory in `checkWriteAndCommit`.